CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 - resolved cookstyle error: libraries/user.rb:168:17 convention: `Layout/IndentationConsistency`
 - resolved cookstyle error: libraries/user.rb:169:15 convention: `Layout/IndentationConsistency`
 - resolved cookstyle error: libraries/user.rb:169:17 convention: `Layout/IndentationConsistency`
+- resolved regression in custom resource names (#272)
```

Comment:
This needs to be moved up to the `## Unreleased` section. Can you please do that?

Reply:
done